### ISSUE TITLE
At 4k resolution, 'Hero names' in the Map becomes smaller and doesnt change even when you max 'Minimap Hero Size' bar.It works perfectly fine when i switch to 1080p resolution, but when i switch to 4k and increase 'Minimap Hero Size' bar, the border color around the text increases, instead of the text. Please fix it.

### ISSUE THREAD

**Issue by an3hul** [ValveSoftware/Dota2-Gameplay#9982]
### Description

It works perfectly fine when i switch to 1080p resolution, but when i switch to 4k to my monitors resolution and increase 'Minimap Hero Size' bar, the border color around the text increases, instead of the text. 

### Screenshots

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/110278524/083c9baf-b751-4b10-8fab-bd0ff3eda72a)

**Comment by Avarise831** [comment]
#8232

**Comment by Ameame1123** [comment]
Yes, all 2160p and 1440p resolution monitors have this problem. The number of pixels of the text on the minimap is still the same as that of 1080p, and it cannot be adjusted. This makes the text on the minimap at 2160p and 1440p resolutions unreadable. You can only use the hero portrait on the minimap instead.
Request valve to fix this issue, which makes me unable to play dota2 with 2160p and 1440p resolution monitors
